cmd/containerd/command/service_windows.go

```diff
@@ -102,6 +107,9 @@ func applyPlatformFlags(context *cli.Context) {
 	} {
 		*v.d = context.GlobalBool(v.name)
 	}
+	if s := context.GlobalString("log-file"); s != "" {
```

Comment:
I think `GlobalString` will return `""` if the param is not set, so you can probably just write `logFileFlag = context.GlobalString("log-file")`.

Reply:
Done. Also enhanced the service name flag handling a little bit.